docs/conf.py

```diff
@@ -167,7 +167,17 @@ def setup(app):
 # -- Options for intersphinx extension ----------------------------------------
 
 intersphinx_mapping = {
-    'python': ('http://docs.python.org/2', None),
-    'pykka': ('http://www.pykka.org/en/latest/', None),
+    'python': ('https://docs.python.org/2', None),
+    'pykka': ('https://www.pykka.org/en/latest/', None),
     'tornado': ('http://www.tornadoweb.org/en/stable/', None),
 }
+
+# -- Options for linkcheck builder -------------------------------------------
+
+linkcheck_ignore = [ #Some sites work in browser but linkcheck fails.
```

Comment:
I believe the flake8 linter will complain about the start of the comment here. You should have two spaces before `#` and one space after.

Same five lines down.